bin.js

```diff
@@ -50,16 +50,29 @@ function degit(dir) {
 
 async function patch(dir) {
   try {
-    if (!fs.existsSync(join(dir, 'src/server.js'))) {
+    const srcDir = fs.readdirSync(join(dir, 'src'))
+    const serverFile = srcDir.find((file) => file.includes('server'))
+
+    if (!serverFile) {
```

Comment:
This is too broad, I would prefer to check exactly for server.js and server.ts with `fs.existsSync`

Something along these lines:
```suggestion
    const serverFile = fs.existsSync(join(dir, 'src/server.js'))
      ? 'src/server.js'
      : fs.existsSync(join(dir, 'src/server.ts'))
      ? 'src/server.ts'
      : ''
```
Also consider including "src/" in the serverFile string so it repeats less often.

Reply:
I don't like the repetitiveness of that part, but it is a more accurate check indeed.

`src/` is now included as part of `serverFile` variable as requested.